src/debian/10/helix/amd64/Dockerfile

```diff
@@ -41,6 +42,15 @@ RUN ln -sf /usr/bin/python3 /usr/bin/python && \
     pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
     pip install ./helix_scripts-*-py3-none-any.whl
 
+# Add MsQuic
+COPY  microsoft.asc /tmp
+RUN apt-key add /tmp/microsoft.asc \
+    && rm /tmp/microsoft.asc \
+    && apt-add-repository https://packages.microsoft.com/debian/10/prod \
+    && apt-get update \
+    && apt-get install -y libmsquic \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
This problem with that is that it is vulnerable to chain attacks since there is no verification if the downloaded key is the correct one. To do that correctly, we would need to verify key fingerprint before using as trusted. That can be done but it takes extra steps. I can do that if you really want to avoid establishing trust via local key file @mthalman 

My preference would be to make singe file for all containers but I did not figure out how to do that. 
Since this is ASCII, we can possibly create it via echo/printf as well.

Reply:
Ok, this is fine. But if there's a vulnerable aspect to the official installation instructions to customers, we should probably get that addressed, yes?

Reply:
yes, there were also some errors for some distributions and we should perhaps update notes to include current/latest OS versions as well. I will take a look.